docs/manual/install/ubuntu.rst

```diff
@@ -2,9 +2,17 @@
 Installing on Debian or Ubuntu
 ==============================
 
+To install the latest stable version of Qtile on Ubuntu (15.10 to 16.10 or newer than 20.04) and Debian 9:
```

Comment:
What's the situation for versions between 16.10 and 20.04?

Reply:
It's in the official repos.
I haven't changed that part.

Reply:
Ah yes, I missed that.

Reply:
Wouldn't we treat those as unsupported now? 16.04 LTS support ended in 2021.

Reply:
> Wouldn't we treat those as unsupported now? 16.04 LTS support ended in 2021.

So, should I remove that part?

Reply:
I would, yes.